File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/OnReachedItemAtPosition.java
/**
 * Copyright © 2017 Wayfair. All rights reserved.
 */
package com.wayfair.brickkit;

/**
 * Callback interface used in {@link BrickRecyclerAdapter} when binding items.
 */
public interface OnReachedItemAtPosition {
    /**
     * Callback from {@link BrickRecyclerAdapter} when binding items.
     *
     * @param position position of item that was bound
     */
    void bindingItemAtPosition(int position);
}

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/OnReachedItemAtPosition.kt
/*
 * Copyright © 2017-2021 Wayfair. All rights reserved.
 */
package com.wayfair.brickkit

/**
 * Callback interface used to notify when binding items.
 */
interface OnReachedItemAtPosition {
    /**
     * Callback when binding items.
     *
     * @param position position of item that was bound
     */
    fun bindingItemAtPosition(position: Int)
}
